status.go

```diff
@@ -186,28 +186,28 @@ func (c *statusController) statusAvailable() error {
 			Type:               osconfigv1.OperatorAvailable,
 			Status:             osconfigv1.ConditionTrue,
 			LastTransitionTime: metav1.Now(),
-			Reason:             "",
+			Reason:             "AsExpected",
```

Comment:
thanks @wking!
can we please move this into a constant? Then in the test for each condition we do something like

```
Expect(v1helpers.FindStatusCondition(co.Status.Conditions, osconfigv1.OperatorAvailable).Reason).To(Equal(reasonAsExpected))```

Reply:
> ...can we please move this into a constant? Then in the tests for each condition...

Done with 9ec8de63 -> d86f36a6.